### PR TITLE
fix(FEC-9534): changing the bitrate post completion of any content changes replay to play icon

### PR DESCRIPTION
**Pull request by RoyBregman**
### Description of the Changes

to fix bug in hls that seek event is fired when changing qualities - if state was idle (i.e end of playback) and seeked event fired don't change state to paused if current time is at the end of the playback

solves FEC-9534

### CheckLists

- [X] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [X] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/player.js [hunk 1]
```diff
@@ -1701,6 +1720,10 @@ export default class Player extends FakeEventTarget {
         if (browser === 'Edge' || browser === 'IE') {
           this._removeTextCuePatch();
         }
+        if (this._engine && this._pendingSelectedVideoTrack) {
+          this._engine.selectVideoTrack(this._pendingSelectedVideoTrack);
```
Comment (yairans) [inline src/player.js]: clearer to do this on `PLAY` or `PLAYING` no?

File: src/player.js [hunk 2]
```diff
@@ -1130,7 +1145,11 @@ export default class Player extends FakeEventTarget {
   selectTrack(track: ?Track): void {
     if (this._engine) {
       if (track instanceof VideoTrack) {
-        this._engine.selectVideoTrack(track);
+        if (this._playbackEnded) {
+          this._pendingSelectedVideoTrack = track;
```
Comment (yairans) [inline src/player.js]: reset on reset

File: src/player.js (hunk 1 above)
Comment (RoyBregman) [reply src/player.js]: done

File: src/player.js (hunk 2 above)
Comment (RoyBregman) [reply src/player.js]: done

**Pull request merged**